services/movie_session.py

```diff
@@ -42,3 +42,10 @@ def update_movie_session(
 
 def delete_movie_session_by_id(session_id: int) -> None:
     MovieSession.objects.get(id=session_id).delete()
+
+
+def get_taken_seats(movie_session_id: int) -> list[dict]:
+    session = MovieSession.objects.get(id=movie_session_id)
```

Comment:
You can use get_movie_session_by_id function here in order to not repeat yourself

Reply:
Yeah, it will be better!) Thx:)